client/src/store/home.ts

```diff
@@ -1,72 +1,13 @@
-'use client';
+import { atom } from 'jotai';
+import { parseAsArrayOf, useQueryStates, parseAsString, useQueryState } from 'nuqs';
 
-import { array, string } from '@recoiljs/refine';
-import { atom, selectorFamily } from 'recoil';
-import { urlSyncEffect } from 'recoil-sync';
+export const useCategoryAtom = () => useQueryState('category', parseAsString);
```

Comment:
I don't like you use the term 'atom' when you use `useQueryState`. Then, when you use it across the app I can't differenciate between the ones that will be stored in the url and the ones that won't. What we have done in other projects is to name these ones as `useSyncCategory` for example.

I like also useUrlCategory, etc...